org.nextcloud.Nextcloud.appdata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You need to install this file manually. Look at RetroArch or Spotify (on Flathub) for examples.